paper.tex

```diff
@@ -81,9 +81,53 @@ \section{Method}
+It can be seen in fig. \ref{fig:stable_diffusion}, that only the nahrawy evaluator has a little bit of red in it but also it is the only one with the output of 'human-human'. Compared to the dall-E generator in fig. \ref{fig:dallE} the amount of green is relatively low. There are also other differences as here only umm-maybe achieved 'human-ai' but also 'human-human'. 
+It is visible that the gaussian noise image processor as well as the salt-and-pepper one are the best processors. This can be seen as they alone achieve a great result as well as their combinations with other ones. It is also worth mentioning that these result are only achieved when using the nahrawy evaluator. Whereas resnet18 which can be seen in both figures has only the output of 'ai-ai'.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=\linewidth]{images/stable-diffusion.png}
+	\caption{Stable Diffusion with different processors and nahrawy, resnet18 and umm-maybe as evaluators}
+	\label{fig:stable_diffusion}
+\end{figure}
+
+\begin{figure}
+	\centering
+	\includegraphics[width=\linewidth]{images/dalle.png}
+	\caption{Dall-E with different processors and nahrawy, resnet18 and umm-maybe as evaluators}
+	\label{fig:dallE}
+\end{figure}
+
+
+In fig. \ref{fig:gpt2} the text processors can be seen. The colors stay the same and the concept as well, where the left bar is using the radar evaluator and the right one the roberta base one. It is visible that the double-whitespace processor alone has little effect whereas in combination with the typo one the result rise enourmously. This is due to the typo processor, since it alone also achieve great results. Here is also one evaluator, which produces only a black bar. 
+Pretty much the exact same results can be seen in fig. \ref{fig:falcon} where radar has just black bars and typo as well as its combination with the roberta base evaluator achieve great results. The difference is that with the falcon generator and the roberta base evaluator in combination, some texts are detected as human made even before processing, which explains the red and grey parts. 
+
+\begin{figure}
+	\centering
+	\includegraphics[width=\linewidth]{images/gpt2.png}
+	\caption{GPT2 with different processors and nahrawy, resnet18 and umm-maybe as evaluators}
+	\label{fig:gpt2}
+\end{figure}
+
+\begin{figure}
+	\centering
+	\includegraphics[width=\linewidth]{images/falcon.png}
+	\caption{Falcon with different processors and nahrawy, resnet18 and umm-maybe as evaluators}
+	\label{fig:falcon}
+\end{figure}
+
+
+All in all can be said that the combination between generators and evaluator plays a huge role in the results. In case of the text generators the difference seems minor, but for example just the fact, that the roberta base evaluator detected around ten falcon generated texts as human made in the middle case of fig. \ref{fig:falcon} and after processing as well, which means that if that would not have been the case, we would have a perfect score here. 
```

Comment:
```suggestion
To summarize we can say, that the combination between generators and evaluators plays a huge role in the results. In case of the text generators the difference seems minor, but for example the roberta base evaluator detected around ten falcon generated texts as human made in the middle case of fig. \ref{fig:falcon} and after processing as well, which means that if that would not have been the case, we would have a perfect score here. 
```